formats/polus-tabular-to-feather-plugin/README.md

```diff
@@ -1,11 +1,12 @@
+
 # Tabular to Feather (v0.1.2)
 This WIPP plugin allows analysts to convert various file formats received by researchers into Arrow Feather File Format (V2). This plugin supports the following file extensions: 
-    - `.fcs`
-    - `.csv`
-    - `.hdf5`
-    - `.fits`
-    - `.parquet`
-    - `.arrow`
+    - `.*.fcs`
+    - `.*.csv`
+    - `.*.hdf5`
+    - `.*.fits`
+    - `.*.parquet`
+    - `.*.arrow`
```

Comment:
These are not going to be rendered as a list in Markdown because they are preceded by a tab.